website/docs/experimental/callbacks.md

```diff
@@ -5,7 +5,9 @@ sidebar_label: Callbacks
 ---
 
 import GithubLink from "@site/src/components/GithubLink"
+import {ExampleGithubLink} from "@site/src/components/GithubLink"
 
+<ExampleGithubLink text="Examples" to="hydra/experimental"/>
```

Comment:
I think users might be confused when this "Examples" button redirects to the `hydra/experimental` directory; there are some things in that directory that are unrelated to callbacks.

What if we create a file `hydra/experimental/callbacks.py` that contains all the callback implementations (instead of using separate files `log_job_return_callback.py` and `pickle_job_info_callback.py`)? The `ExampleGithubLink` in the docs could then point to that specific file `callbacks.py`.
This would make the yaml files simpler too: users would write `_target_: hydra.experimental.callbacks.LogJobReturnCallback` instead of `_target_: hydra.experimental.log_job_return_callback.LogJobReturnCallback`).

Reply:
That's a great idea! Since this would mean touching the PickleJobCallback as well, I will create that PR as a follow up. 
For now I will remove this change.